microcosm_postgres/health.py

```diff
@@ -6,4 +6,20 @@
 
 
 def check_health(graph):
+    """
+    Basic database health check.
+
+    """
     SessionContext.session.execute("SELECT 1;")
+
+
+def check_alembic(graph):
```

Comment:
Are those `check` functions called by default by the `health_convention`, where `check_foo` will give a `"foo"` key in the response json?